apps/etterlatte-saksbehandling-ui/client/src/components/behandling/soeknadsoversikt/viderefoere-opphoer/ViderefoereOpphoer.tsx

```diff
@@ -30,20 +44,35 @@ export const ViderefoereOpphoer = ({
         </BodyShort>
       </VStack>
       <Box paddingInline="3 0" minWidth="18.75rem" width="10rem" borderWidth="0 0 0 2" borderColor="border-subtle">
-        {vurdert && (
-          <ViderefoereOpphoerVurdering
-            virkningstidspunkt={behandling.virkningstidspunkt ? new Date(behandling.virkningstidspunkt.dato) : null}
-            viderefoertOpphoer={behandling.viderefoertOpphoer}
-            redigerbar={redigerbar}
-            setVurdert={(visVurderingKnapp: boolean) => setVurdert(visVurderingKnapp)}
+        {visVurdering ? (
```

Comment:
Kanskje den burde hete `virkningstidspunktErSatt`, for er jo det som styrer om alerten skal vises eller ikke

Reply:
Gjort litt endringer her